broadway_api/flags.py

```diff
@@ -0,0 +1,124 @@
+from flagset import Flag
+
+app_flags = {
+    "debug": Flag(
+        bool,
+        default=False,
+        cmdline_name=["-d", "--debug"],
+        env_name="BROADWAY_DEBUG",
+        config_name="debug",
+        help="enable debug mode",
+    ),
+    "token": Flag(
+        str,
+        required=True,
```

Comment:
Can we make this optional? if not provided, we generate a random one and use it (and print it).